shared/js/background/events.es6.js

```diff
@@ -352,8 +366,13 @@ browser.runtime.onMessage.addListener((req, sender) => {
 /**
  * Fingerprint Protection
  */
-// TODO fix for manifest v3
-let sessionKey = getHash()
+let sessionKey
```

Comment:
How about instead we make the `browser.runtime.onMessage` listener `async` and do `await getSessionKey()` where we need the session key? Then we could avoid having the local variable + init function.

I don't think this code is susceptible to race conditions, but doing it that way would also be setting a better example of how to manage state with MV3 (e.g. using session storage and waiting for it to be ready, rather than using local variables). That might help others avoid foot-guns in the future?

Reply:
Making the onMessage handler be async is a very large task, we should avoid that here (mostly why I've avoided this feedback as didn't have chance to fix it).

I've moved the one handler into the message-handlers file instead and made this async; I think this matches the spirit of your feedback and removes the local var.